pinot-tools/src/main/java/com/linkedin/pinot/tools/perf/PerfBenchmarkDriver.java

```diff
@@ -308,49 +305,26 @@ public void addSegment(SegmentMetadata segmentMetadata) {
   public static void waitForExternalViewUpdate(String zkAddress, final String clusterName, long timeoutInMilliseconds) {
     final ZKHelixAdmin helixAdmin = new ZKHelixAdmin(zkAddress);
 
-    /* Use one of these three classes instead of deprecated ClusterStateVerifier.
-    BestPossibleExternalViewVerifier
-    StrictMatchExternalViewVerifier
-    ClusterLiveNodesVerifier
-    */
-
-    org.apache.helix.tools.ClusterVerifiers.ClusterStateVerifier.Verifier customVerifier = new org.apache.helix.tools.ClusterVerifiers.ClusterStateVerifier.Verifier() {
-
-      @Override
-      public boolean verify() {
-        List<String> resourcesInCluster = helixAdmin.getResourcesInCluster(clusterName);
-        LOGGER.info("Waiting for the cluster to be set up and indexes to be loaded on the servers" + new Timestamp(
-            System.currentTimeMillis()));
-        for (String resourceName : resourcesInCluster) {
-          // Only check table resources and broker resource
-          if (!TableNameBuilder.isTableResource(resourceName) && !resourceName.equals(
-              CommonConstants.Helix.BROKER_RESOURCE_INSTANCE)) {
-            continue;
-          }
-
-          IdealState idealState = helixAdmin.getResourceIdealState(clusterName, resourceName);
-          ExternalView externalView = helixAdmin.getResourceExternalView(clusterName, resourceName);
-          if (idealState == null || externalView == null) {
-            return false;
-          }
-          Set<String> partitionSet = idealState.getPartitionSet();
-          for (String partition : partitionSet) {
-            Map<String, String> instanceStateMapIS = idealState.getInstanceStateMap(partition);
-            Map<String, String> instanceStateMapEV = externalView.getStateMap(partition);
-            if (instanceStateMapIS == null || instanceStateMapEV == null) {
-              return false;
-            }
-            if (!instanceStateMapIS.equals(instanceStateMapEV)) {
-              return false;
-            }
-          }
-        }
-        LOGGER.info("Cluster is ready to serve queries");
-        return true;
+    List<String> allResourcesInCluster = helixAdmin.getResourcesInCluster(clusterName);
+    Set<String> tableAndBrokerResources = new HashSet<>();
+    for (String resourceName : allResourcesInCluster) {
+      // Only check table resources and broker resource
+      if (!TableNameBuilder.isTableResource(resourceName) && !resourceName.equals(
```

Comment:
(OCD) Maybe better to change to:
if (TableNameBuilder.isTableResource(resourceName) || resourceName.equals(CommonConstants.Helix.BROKER_RESOURCE_INSTANCE)) {
  tableAndBrokerResources.add(resourceName);
}